src/Integration.Vsix/Commands/ProjectSonarLintMenuCommand.cs

```diff
@@ -51,12 +52,9 @@ protected override void InvokeInternal()
 
         protected override void QueryStatusInternal(OleMenuCommand command)
         {
+            Debug.Assert(this.propertyManager != null, "Property manager should not be null");
             command.Enabled = false;
             command.Visible = false;
-            if (this.propertyManager == null)
```

Comment:
This null check is unnecessary (and untestable) - the constructor throws if the value is null.